src/Microsoft.AspNetCore.Mvc.Core/Internal/DelegatingBinder.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Internal
+{
+    public class DelegatingBinder : IModelBinder
```

Comment:
Doesn't need `///` comments but merits _some_ explanation w/in the class. Might mention that it initially serves as a placeholder but will be resolved prior to binder execution. And, when executing, delegates to the `Inner` binder.

Separately `DelegatingBinder` is a slightly odd name because, while delegation is what it does later on, "placeholdering" is why it exists and a more important role. Suggest `BinderPlaceholder`.